pkg/cloudprovider/cloudprovider.go

```diff
@@ -358,6 +358,7 @@ func (c *CloudProvider) instanceToNodeClaim(i *instance.Instance, instanceType *
 	}
 	labels[v1.LabelTopologyZone] = i.Zone
 	labels[corev1beta1.CapacityTypeLabelKey] = i.CapacityType
+	labels[v1alpha1.LabelInstanceAMIID] = i.ImageID
```

Comment:
```suggestion
	labels[v1beta1.LabelInstanceAMIID] = i.ImageID
```

We should add this to the `beta` APIs package since this is what we are taking forward